### ISSUE TITLE
updating babel packages results in test errors

### ISSUE THREAD

**Issue by ericblade** [ericblade/quagga2#13]
npm-upgrade babel* results in fairly significant appearing test failures.  babel has apparently undergone some restructuring, and updating may? be difficult.

**Comment by ericblade** [comment]
@domratchev 's patch here https://github.com/serratus/quaggaJS/pull/385/commits/05e6b46121fb2ecb3c0c971637738ae47807fa19

contributes a large chunk of updates that include updating babel, particularly the .babelrc at the top, 
and the following section of package.json:

````
    "@babel/core": "^7.5.5",
    "@babel/plugin-proposal-class-properties": "^7.5.5",
    "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
    "@babel/plugin-transform-runtime": "^7.5.5",
    "@babel/preset-env": "^7.5.5",
    "@babel/preset-typescript": "^7.3.3",
    "@types/async": "^3.0.1",
    "@types/node": "^12.6.8",
    "@types/sinon": "^7.0.13",
    "async": "^3.1.0",
    "babel-loader": "^8.0.6",
    "babel-plugin-add-module-exports": "^1.0.2",
    "babel-plugin-istanbul": "^5.2.0",
````

**Comment by ericblade** [comment]
fixed in PR#24
